model/config.go

```diff
@@ -2887,6 +2887,43 @@ func (s *GlobalRelayMessageExportSettings) SetDefaults() {
 	}
 }
 
+type WranglerSettings struct {
+	PermittedWranglerUsers                   []string
+	AllowedEmailDomain                       []string
+	MoveThreadMaxCount                       int64
+	MoveThreadToAnotherTeamEnable            bool
+	MoveThreadFromPrivateChannelEnable       bool
+	MoveThreadFromDirectMessageChannelEnable bool
+	MoveThreadFromGroupMessageChannelEnable  bool
+}
+
+func (w *WranglerSettings) SetDefaults() {
+	if w.PermittedWranglerUsers == nil {
+		permittedWranglerUsers := []string{}
+		w.PermittedWranglerUsers = permittedWranglerUsers
```

Comment:
Alternatively, I think we could write:
```suggestion
w.PermittedWranglerUsers = make([]string, 0)
```